.prettierignore

```diff
@@ -4,20 +4,19 @@ __pycache__/
 .cargo/
 .github/
 .vscode/
-compiler/
-jupyterlab/.pytest_cache/
-jupyterlab/lib/
-jupyterlab/.venv/
-jupyterlab/qsharp_jupyterlab/labextension/
-library/
-npm/dist/
-npm/lib/
-npm/src/*.generated.ts
-pip/.venv/
-pip/.pytest_cache/
-pip/src/**/*.html
-playground/public/libs/
-samples/
-target/
-vscode/out/
-wasm/
+.venv/
+/compiler/
```

Comment:
`/` will make sure to match paths only starting at repo root. Noticed this when prettier started ignoring files under the new `npm/src/compiler/` subfolder I added.